.pre-commit-config.yaml

```
@@ -28,14 +28,14 @@ repos:
     - id: text-unicode-replacement-char
 
 - repo: https://github.com/asottile/pyupgrade
-  rev: v2.38.2
+  rev: v3.1.0
   hooks:
     - id: pyupgrade
       args: ["--py38-plus"]
       exclude: "asdf/extern/.*"
 
 - repo: https://github.com/PyCQA/autoflake
-  rev: v1.6.1
+  rev: v1.7.2
```

```suggestion
  rev: v1.7.3
```

Should fix auto flake.